pkgs/development/python-modules/tensorpack/default.nix

```
@@ -0,0 +1,52 @@
+{ lib, buildPythonPackage, setuptools_scm, fetchPypi, isPy3k
+, numpy
+, six
+, termcolor
+, tabulate
+, tqdm
+, msgpack
+, msgpack-numpy
+, pyzmq
+, psutil
+, scikitimage
+, flake8
+, tensorflow
+}:
+
+buildPythonPackage rec {
+  pname = "tensorpack";
+  version = "0.9.8";
+  disabled=!isPy3k; # Python 2 could be added by motivated contributer
```

or not https://github.com/tensorpack/tensorpack/commit/0641618d311bd358abef7e864afe1868d97c1f14
```suggestion
  disabled = !isPy3k;
```